modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/entries/frontend/data/set/data/provider/RelatedModelsFDSDataProvider.java

```diff
@@ -78,14 +80,41 @@ public List<RelatedModel> getItems(
 		long objectEntryId = ParamUtil.getLong(
 			httpServletRequest, "objectEntryId");
 
+		if (objectDefinition.isSystem()) {
+			return TransformUtil.transform(
+				objectRelatedModelsProvider.getSystemRelatedModels(
+					objectEntryId, objectDefinition, objectRelationship),
+				entryMap -> {
+					Long titleFieldId =
+						objectDefinition.getTitleObjectFieldId();
+
+					String titleFieldName =
+						objectDefinition.getPKObjectFieldDBColumnName();
+
+					if (titleFieldId > 0) {
+						ObjectField objectField =
+							_objectFieldLocalService.getObjectField(
+								titleFieldId);
+
+						titleFieldName = objectField.getDBColumnName();
+					}
+
+					return new RelatedModel(
+						(Long)entryMap.get(
```

Comment:
Hey @p-albuquerque,
Did you try to replace (Long) to GetterUtil.getLong()? Does it work?